mne/viz/_3d.py

```diff
@@ -1143,3 +1143,107 @@ def plot_dipole_locations(dipoles, trans, subject, subjects_dir=None,
         fig.scene.x_plus_view()
 
     return fig
+
+
+def snapshot_montage(fig, montage, with_points=False):
+    """Take a snapshot of a Mayavi Scene and project 3d points to 2d coords.
+
+    Parameters
+    ----------
+    fig : instance of Mayavi Scene
+        The figure on which you've plotted electrodes using `plot_trans`.
+    montage : instance of DigMontage
+        The digital montage for the electrodes plotted in the scene.
+    with_points : bool
```

Comment:
I think that'd work as well, yes. Though it'd be a little bit clunkier since you'd need to make the plot w/ sensors, then rotate it to the angle you want and record the view, then make the plot again, set the view, and take the snapshot. As opposed to here you'd just call `plot_trans`, rotate it, then call `snapshot_montage`.